test/exq_scheduler_test.exs

```diff
@@ -19,4 +28,104 @@ defmodule ExqSchedulerTest do
     :timer.sleep(2000)
     assert_job_uniqueness()
   end
+
+  @tag config: configure_env(env(), 1500, 1000,[schedule_cron_1h: %{
+                                                 "cron" => "0 * * * * *",
+                                                 "class" => "QWorker",
+                                                 "queue" => "SuperQ",
+                                                 "include_metadata" => true}])
+  test "check continuity" do
+    redis = redis_pid("continuity")
+    storage_opts = Storage.build_opts(env([:redis, :name], redis))
+    :timer.sleep(4000) # 3+ Hour
+
+    jobs = get_jobs_from_storage(redis, Storage.queue_key("default", storage_opts))
```

Comment:
This should probably be moved to test helper, pass the queue name and class use, use the redis that's used for flushdb now.